TS/translations/ru_RU/ts.po

```diff
@@ -0,0 +1,1595 @@
+msgid "Image Resize"
+msgstr "Размер изображения изменён"
+
+msgid "Import file from"
+msgstr "&Импортировать файл из"
+
+msgid "Increase column width (ALT+mousewheel)\tALT+PGUP"
+msgstr "У&величить Ширину Столбца (ALT+mousewheel)\tALT+PGUP"
+
+msgid "Increase column width (no sub grids)\tCTRL+ALT+PGUP"
+msgstr "У&величить Ширину Столбца (без подсеток)\tCTRL+ALT+PGUP"
+
+# & - не в том месте!..
+msgid "Indented &Text..."
+msgstr "&Текст с Отступами..."
+
+msgid "Indented text..."
+msgstr "&Текст с Отступами..."
+
+# Сообщения
+msgid "Initialization Error"
+msgstr "Ошибка инициализации"
+
+# Сообщения
+msgid "Internal error: unimplemented operation!"
+msgstr "Внутренняя ошибка: невыполненная операция!"
+
+# Сообщения
+msgid ""
+"It seems that you are about to drop a TreeSheets file. What would you like "
+"to do?"
+msgstr ""
+"Кажется, вы собираетесь сбросить файл TreeSheet. Что бы вы собираетксь "
+"сделать?"
+
+# Сообщения
+msgid "JPEG file is corrupted!"
+msgstr "JPEG-файл поврежден!"
+
+# Сообщения
+msgid "Key binding"
+msgstr "Привязка клавиш"
+
+# Сообщения
+msgid "Keybinding cancelled."
+msgstr "Привязка клавиш отменена."
+
+msgid "Line Style Rendering\tALT+9"
+msgstr "... в стиле &Линии\tALT+9"
+
+msgid "Load interactive &tutorial...\tF1"
+msgstr "Загрузить интерактивный &Учебник...\tF1"
+
+msgid "Load operation reference...\tCTRL+ALT+F1"
+msgstr "Загрузить Справочник Операций...\tCTRL+ALT+F1"
+
+# Сообщения
+#, c-format
+msgid "Loaded %s (%d cells, %d characters) in %d milliseconds."
+msgstr "Загружено %s (%d ячеек, %d символов) за %d миллисекунд."
+
+# Сообщения
+msgid ""
+"Make a 1xN selection to indicate which column to sort on, and which rows to "
+"affect"
+msgstr ""
+"Сделайте выборку 1xN, чтобы указать, по какому столбцу производить "
+"сортировку и в каких строках"
+
+# Сообщения
+msgid "Make a hierarchy layout more vertical (default) or more horizontal"
+msgstr ""
+"Сделать иерархическую компоновку более вертикальной (по умолчанию) или более "
+"горизонтальной"
+
+# Сообщения
+msgid "Maximum number of launches reached."
+msgstr "Достигнуто максимальное количество запусков."
+
+msgid "Minimize on close"
+msgstr "Свернуть при закрытии"
+
+msgid "Minimize to tray"
+msgstr "Свернуть в Трей"
+
+# Сообщения
+msgid "More than one cell must be selected."
+msgstr "Необходимо выбрать более одной ячейки."
+
+msgid "Move Cells Down\tCTRL+DOWN"
+msgstr "... вНиз\tCTRL+DOWN"
+
+msgid "Move Cells Left\tCTRL+LEFT"
+msgstr "Переместить Ячейки вЛево\tCTRL+LEFT"
+
+msgid "Move Cells Right\tCTRL+RIGHT"
+msgstr "... вПраво\tCTRL+RIGHT"
+
+msgid "Move Cells Up\tCTRL+UP"
+msgstr "... вВерх\tCTRL+UP"
+
+msgid "Move Selection Down\tDOWN"
+msgstr "... вНиз\tDOWN"
+
+msgid "Move Selection Left\tLEFT"
+msgstr "Переместить Курсор Клетки вЛево\tLEFT"
+
+msgid "Move Selection Right\tRIGHT"
+msgstr "... вПраво\tRIGHT"
+
+msgid "Move Selection Up\tUP"
+msgstr "... вВерх\tUP"
+
+msgid "Move to next cell\tTAB"
+msgstr "Перейти к Следующей Ячейке\tTAB"
```

Comment:
This seems fine to me, no idea why `TAB` wouldn't show.. maybe a wxWidgets issue.